joplin/js/CreateContentModal/index.js

```diff
@@ -13,6 +13,7 @@ import ButtonBar from './ButtonBar.js';
 import './index.scss';
 
 const MAX_TITLE_LENGTH = 58;
+const DEPARTMENT_LIST = window.departments;
```

Comment:
Why do we need to use a global variable here?

Reply:
not sure we do, I was looking at what we did last time we had departments in the modal, and I think I remember having issues using window later on but I could be mistaken